### PR TITLE
request for help... Adding CCE check to verify references

### PR DESCRIPTION
**Pull request by shawndwells**
Attempting to modify verify-references to make https://github.com/OpenSCAP/scap-security-guide/issues/1281 easier. I'm going crazy and could use some help!

If I pass --rules-without-cce, everything works as expected
`````
$ /var/www/html/scap-security-guide/shared/utils/verify-references.py --rules-without-cce ssg-rhel7-xccdf.xml 
No CCE assigned to XCCDF Rule: audit_rules_mac_modification
.....
`````

And if I add the CCE for audit_rules_mac_modification back in, the finding disappears. Success!

However, if I attempt with --all-checks, I get nothing!
`````
$ /var/www/html/scap-security-guide/shared/utils/verify-references.py --all-checks ssg-rhel7-xccdf.xml
`````

rules_without_cce is defined to run during all checks, but no dice:
`````if options.rules_without_cce or options.all_checks:`````

Any tips?

### CODE REVIEW

File: shared/utils/verify-references.py [hunk 1]
```diff
@@ -205,6 +209,22 @@ def main():
                 print "No severity assigned to XCCDF Rule: " + rule.get("id")
                 exit_value = 1
 
+    if options.rules_without_cce or options.all_checks:
+        for rule in rules:
+		# find all CCEs in current rule
+		cces = rule.findall(".//{%s}ident" % xccdf_ns)
```
Comment (iankko) [inline shared/utils/verify-references.py]: This should be enhanced like follows:
```
cces = rule.findall(".//{%s}ident[@system=\"%s\"]" % (xccdf_ns, cce_ref_href))
```
to search just CCE ```ident``` elements. (The current version is searching all ```ident``` elements, thus e.g. also something like:
```
<ident system="http://iase.disa.mil/stigs/os/unix-linux/Pages/index.aspx">DISA FSO RHEL-06-000013</ident>
```
will match the condition, and ```cces``` won't be None. Thus no warning will be printed for that rule.

**Comment by redhatrises** [comment]
@shawndwells do you have the `ssg-rhel7-oval.xml` file in the same location as the `ssg-rhel7-xccdf.xml` file, or are you running this out of the `RHEL/7/output` directory?

I ran it successfully with the `--all-checks` by either running from the `RHEL/7/output` directory or have both the `ssg-rhel7-xccdf.xml` and `ssg-rhel7-oval.xml` in the same directory.

File: shared/utils/verify-references.py (hunk 1 above)
Comment (shawndwells) [inline shared/utils/verify-references.py]: While perhaps more efficient, that results in a traceback:
`````
Traceback (most recent call last):
  File "/var/www/html/scap-security-guide/shared/utils/verify-references.py", line 295, in <module>
    main()
  File "/var/www/html/scap-security-guide/shared/utils/verify-references.py", line 215, in main
    cces = rule.findall(".//{%s}ident[@system=\"%s\"]" % xccdf_ns, cce_ref_href)
TypeError: not enough arguments for format string
`````

Comment (mpreisler) [inline shared/utils/verify-references.py]: It's missing the brackets.

**Pull request closed without merging**